beekeeper-api/src/main/java/com/expediagroup/beekeeper/api/response/PathResponseConverter.java

```diff
@@ -56,7 +56,9 @@ public static Page<HousekeepingPathResponse> convertToHousekeepingPathResponsePa
       HousekeepingPathResponse housekeepingPathResponse = convertToHousekeepingPathResponse(housekeepingPath);
       housekeepingPathResponseList.add(housekeepingPathResponse);
     }
-    return new PageImpl<>(housekeepingPathResponseList);
+    PageImpl<HousekeepingPathResponse> HousekeepingPathResponses = new PageImpl<>(
```

Comment:
```suggestion
    PageImpl<HousekeepingPathResponse> housekeepingPathResponses = new PageImpl<>(
```

Reply:
I have committed this suggestion but also updated line 61 

from:
`return HousekeepingPathResponses;`
to:
`return housekeepingPathResponses;`

This is now correct and working properly